src/objectTranslations/Schedulable__c-de.objectTranslation

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <caseType>Nominative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Nominative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Accusative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Accusative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Genitive</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Genitive</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Dative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Dative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <fields>
+        <help>Legt fest, ob der Auftrag aktiv ist</help>
+        <label>Aktiv</label>
+        <name>Active__c</name>
+    </fields>
+    <fields>
+        <help>Klasse, deren Ausführung mit einer bestimmten Häufigkeit geplant wird</help>
+        <label>Klassenname</label>
+        <name>Class_Name__c</name>
+    </fields>
+    <fields>
+        <help>Wie oft der Auftrag ausgeführt wird</help>
+        <label>Häufigkeit</label>
+        <name>Frequency__c</name>
+        <picklistValues>
+            <masterLabel>Daily</masterLabel>
+            <translation><!-- Daily --></translation>
```

Comment:
I based the [filter list](https://github.com/SalesforceFoundation/Cumulus/blob/dev/scripts/stf/filters.json) on [the spreadsheet](https://docs.google.com/spreadsheets/d/1Z9HsXtt4ioIVFm1D5CAnLcWWYSMgxCzeeJnYVBtVubI) where Kevin reviewed each key and pointed out which values might be problematic if translated.  I, myself, have never investigated the reasons why or done any experimentation.

According to Kevin's comment for the Scheduleable__c.Frequency__c field, this value can not be translated [because the value we compare it to in the code is an Enum](https://github.com/SalesforceFoundation/Cumulus/blob/dev/src/classes/UTIL_MasterSchedulableHelper.cls#L88).

Many of the the "untranslatable" fields are documented in the "[Nonprofit Starter Pack and Translation](https://powerofus.force.com/articles/Resource/Nonprofit-Starter-Pack-and-Translation)" knowledge article.

However, Elena recently [posted in the hub](https://powerofus.force.com/0D58000002nmmMB) that she tried translating some of the disallowed values, and she found that the workflow rules relating to these values do still work.  (Our documentation says that translating these values will break workflow rules).

So, I suspect that _some_ of these picklist values actually can be safely translated, since I think that querying a picklist value in SOQL will return the untranslated label, [unless you specify `toLabel()`](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_tolabel.htm) for the field in the query.  The only way to be sure is to look carefully at where each of these values is being queried and used, and to experiment/test to verify the behavior of the system after translation.

I would love it if we invested some time in evaluating which fields are truly untranslatable, and then fix the code so that they are translatable.  As far as I know, there is nothing in the backlog for further investigation in this area.